bin/upgrading/update-db-general.pl

```diff
@@ -3120,6 +3120,31 @@
 )
 EOC
 
+register_tablecreate( "profile_services", <<'EOC');
+CREATE TABLE profile_services (
+    name VARCHAR(40) NOT NULL,
+    userprop VARCHAR(40),
+    imgfile VARCHAR(40) NOT NULL,
+    title_ml VARCHAR(80) NOT NULL,
+    url_format VARCHAR(255),
+    maxlen TINYINT(3) UNSIGNED NOT NULL,
+
+    PRIMARY KEY (name)
+)
+EOC
+
+# clustered
+register_tablecreate( "user_profile_accts", <<'EOC');
+CREATE TABLE user_profile_accts (
+    userid INT(10) UNSIGNED NOT NULL,
+    name VARCHAR(40) NOT NULL,
+    value VARCHAR(255) NOT NULL,
+
+    PRIMARY KEY (userid, name, value),
+    INDEX (userid)
```

Comment:
this is implied by having the primary key (since a primary key is an index), so this line is unnecessary (I don't think it does anything?)